static/app/components/avatar/avatarList.tsx

```diff
@@ -1,14 +1,16 @@
 import {css} from '@emotion/react';
 import styled from '@emotion/styled';
 
+import TeamAvatar from 'sentry/components/avatar/teamAvatar';
 import UserAvatar from 'sentry/components/avatar/userAvatar';
 import {Tooltip} from 'sentry/components/tooltip';
 import {AvatarUser} from 'sentry/types';
+import {TeamParticipant} from 'sentry/types/group';
 
 type UserAvatarProps = React.ComponentProps<typeof UserAvatar>;
 
 type Props = {
-  users: AvatarUser[];
+  members: Array<AvatarUser | TeamParticipant>;
```

Comment:
@roggenkemper should `TeamParticipant` be called `AvatarTeam` to be consistent? I also am not sure about `members` here, another variable name might be better

Reply:
i thought of that, but as of now, I don't believe we actually let teams set their own avatar, so i didn't think the name would make sense. agree on `members` , will try to think of something better